src/Components/CreateImageWizard/steps/Registration/components/Registration.tsx

```diff
@@ -77,11 +77,12 @@ const Registration = () => {
                     <ActivationKeysList />
                   </Content>
                 )}
-              {process.env.IS_ON_PREMISE && (
-                <Content className='pf-v6-u-pb-sm'>
-                  <ManualActivationKey />
-                </Content>
-              )}
+              {process.env.IS_ON_PREMISE &&
+                registrationType.startsWith('register-now') && (
```

Comment:
I changed this part to not show activation key and org id for other options, I hope that's okay. @kingsleyzissou